tpls/gtest/gtest/README

```diff
@@ -1,5 +1,5 @@
-This is a fused source version of gtest 1.7.0. All that should be necessary to
-start using gtest in your package is to declare the dependency and include
+This is a fused source version of gtest (47f819c3). All that should be necessary
```

Comment:
Why did you not use a release version?

Reply:
From [GoogleTest README](https://github.com/google/googletest#live-at-head):

> GoogleTest now follows the Abseil Live at Head philosophy. We recommend using the latest commit in the master branch in your projects.

The [latest release notes](https://github.com/google/googletest/releases/tag/release-1.11.0) also state:

> GoogleTest will not accept patches of new features to v1.11.0.
We recommend building GoogleTest from the latest commit instead.
Exceptional critical bug fixes may be considered.

Reply:
The latest release (v1.11.0) is relatively fresh, so we could use that instead.